test/core.py

```diff
@@ -27,6 +30,10 @@ def test_get_env(self):
 
         self.assertEqual("{'great_variable': 'value', 'another_variable': 'another value'}", str(env))
 
+    def test_date_arg_parsing(self):
+        self.assertEqual(core.get_date("+1y 2m3d"), date.today() + relativedelta(years=1, months=2, days=3))
+        self.assertEqual(core.get_date("-1y2d"), date.today() - relativedelta(years=1, days=2))
+        self.assertEqual(core.get_date("1111-11-11"), date(1111, 11, 11))
```

Comment:
```suggestion
        self.assertEqual(core.get_date("1111-11-13"), date(1111, 11, 13))
```
This way, we make sure the month and the day are correctly parsed (if they happen to be inverted, the date will become invalid and the CI will fail)